testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/SuiteRanger.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.suite.suites;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tests.product.launcher.env.EnvironmentConfig;
+import io.trino.tests.product.launcher.env.environment.EnvMultinodeApacheRanger;
+import io.trino.tests.product.launcher.suite.Suite;
+import io.trino.tests.product.launcher.suite.SuiteTestRun;
+
+import java.util.List;
+
+import static io.trino.tests.product.launcher.suite.SuiteTestRun.testOnEnvironment;
+
+public class SuiteRanger
```

Comment:
I see that your product tests were not executed in CI. In order to execute them you need to add `suite-apache-ranger` (that comes from transforming camel case `SuiteApacheRanger`) to the list in https://github.com/trinodb/trino/blob/0a787abb8ba8f76ee0d96c70349e24a6b8027f2e/.github/workflows/ci.yml#L868.

Reply:
Perhaps a Trino maintainer can help update the CI for this?

Reply:
All you need to is to that a line I described in https://github.com/trinodb/trino/pull/22675#discussion_r1806798179.

Reply:
I was able to modify your branch. Added this line let's see what is the status of the execution.

Reply:
@kokosing  - thank you for your help, which saved few hours of me experimenting with changes and verifying them.